spec/videos/video_spec.rb

```diff
@@ -19,6 +19,7 @@
       it { expect(video.length).to be_a(Float) }
       it { expect(video.picture).to be_a(String)}
       it { expect(video.page_name).to be_a(String) }
+      it { expect(video.page_id).to be_a(String) }
```

Comment:
@bruce-fs Does _every_ Facebook video return a value for this field?

Or some videos return `nil`?